public/app/features/search/service/searcher.ts

```diff
@@ -1,11 +1,15 @@
+import { config } from '@grafana/runtime';
+
 import { BlugeSearcher } from './bluge';
+import { SQLSearcher } from './sql';
 import { GrafanaSearcher } from './types';
 
 let searcher: GrafanaSearcher | undefined = undefined;
 
 export function getGrafanaSearcher(): GrafanaSearcher {
   if (!searcher) {
-    searcher = new BlugeSearcher();
+    const useBluge = config.featureToggles.panelTitleSearch && window.location.search.indexOf('index=sql') < 0;
```

Comment:
@axelavargas -- right now you can test this by adding "&index=sql" to any of the pages.  Under the hood it will swap using the SQL interface for the same component 🎉 

When we are more confident with the UI, we can remove all the other frontend instances of `config.featureToggles.panelTitleSearch`